cpop_bot/music_grabber.py

```diff
@@ -70,14 +72,11 @@ def __get_filesize(self, info_format):
             audio_filesize = urlopen(request_head).headers['Content-Length']
         return audio_filesize
 
-    def __add_downloads_to_info_dict(self, info):
-        webpage_id = info['id']
-        extractor = info['extractor']
+    def __add_downloads_to_info_dict(self, info, ydl_opts):
         thumbnail_url = info['thumbnail']
-        basename = DOWNLOAD_DIR + extractor + '_' + webpage_id
-        thumbnail_file = basename + "." + \
+        audio_file = YoutubeDL(ydl_opts).prepare_filename(info)
+        thumbnail_file = audio_file.rsplit(".", 1)[-2] + "." + \
```

Comment:
why did you have to add this

Reply:
`prepare_filename` will output the filename of the audio file (in this case, it would be something like `/tmp/cpop_bot/Vexento - Masked Heroes - youtube-ZLhfr8mpzxU.webm`

Previous I manually get "extract" "id" "ext" and set the "basename" variable, but I found "prepare_filename" which process info_dict and return the output filename, it's better to be done this way.

`audio_file.rsplit(".", 1)[-2]` will be `/tmp/cpop_bot/Vexento - Masked Heroes - youtube-ZLhfr8mpzxU`, and thumbnail file's path has the same part, and we just append extension of thumbnail to thumbnail_file variable.

Reply:
yeah my bad you already mentioned it in the pull request description 😂

i was just too tired